src/AstGen.zig

```diff
@@ -5000,6 +5000,11 @@ fn orelseCatchExpr(
         if (mem.eql(u8, tree.tokenSlice(payload), "_")) {
             return astgen.failTok(payload, "discard of error capture; omit it instead", .{});
         }
+
+        const payload_name = try astgen.identAsString(payload);
```

Comment:
This is the same as `err_name` below, both should use the same variable. The previous `tokenSlice` could also use the same variable as `payload_str`.

Reply:
Sorry I missed those, I haven't worked on stage2 before so its a lot to learn. Should be sorted now!